_scalingo_api/apps.md

```diff
@@ -718,9 +718,8 @@ executed in the environment of your application.
 * `env` (*object*): Environment variables to inject into the container (additionally to those of your apps)
 * `size` (*string*, default `"M"`): Size of the container (e.g. S, M, etc)
 * `detached` (*boolean*, default `false`): Foreground task by default, set to `true` if the container has to be run in background.
-* `async` (*boolean*, default `false`): Create one-off synchronously by default, set to `true` if you want the one-off to be created asynchronously.
 
-For one-off started with the parameter `async` to `true`:<br>
+For one-off started with the parameter `detached` to `true`:<br>
```

Comment:
It's actually the reverse. We need to have the operation to attached to it and follow progress